driver/bpf/fillers.h

```diff
@@ -2367,6 +2367,7 @@ FILLER(proc_startupdate_3, true)
 	struct clone_args cl_args;
 #endif
 
+	res = PPM_FAILURE_BUG;
```

Comment:
Why here and not at line `2364`?

Reply:
Yep in the bpf probe we can; on kmod we can't.
I respected the kmod c standard even if i hadn't to!

Reply:
Well, above i didn't respect it ahah i will fix this.